examples/hostpath-provisioner/go.mod

```diff
@@ -3,10 +3,11 @@ module sigs.k8s.io/kubernetes-sigs/sig-storage-lib-external-provisioner/examples
 go 1.13
```

Comment:
should this be updated to 1.15?

Reply:
The exact semantic of that field is unclear. My understanding of https://github.com/golang/go/issues/30791 is that it refers to the language features that the code uses and not the compiler version that it supports, so leaving it at 1.13 (or even older) is fine.